src/main/resources/application.yaml

```diff
@@ -19,7 +19,7 @@ spring:
     baselineOnMigrate: true
     fail-on-missing-locations: false
   datasource:
-    url: 'jdbc:postgresql://localhost:5432/hapi_fhir_db'
+    url: "jdbc:postgresql://localhost:5432/hapi_fhir_db"
```

Comment:
Since we are using ENV variables, these values are overridden.  It can be a bit misleading to have these values here.   With that in mind, should we delete url, username, and password properties?